File: docs/code_examples/scalar_simple.py
"""Log a scalar over time."""
import math

import rerun as rr

rr.init("rerun_example_scalar", spawn=True)

for step in range(0, 64):
    rr.set_time_sequence("step", step)
    rr.log("scalar", rr.TimeSeriesScalar(math.sin(step / 10.0)))

File: docs/code-examples/scalar_simple.cpp
// Log a scalar over time.

#include <rerun.hpp>

#include <cmath>

int main() {
    auto rec = rerun::RecordingStream("rerun_example_scalar");
    rec.connect().throw_on_failure();

    for (int step = 0; step < 64; ++step) {
        rec.set_time_sequence("step", step);
        rec.log("scalar", rerun::TimeSeriesScalar(std::sin(static_cast<double>(step) / 10.0)));
    }
}
